src/deploy/functions/prompts.ts

```diff
@@ -174,7 +171,8 @@ export async function promptForMinInstances(
   // Considerations for future versions:
   // Group Tier 1 and Tier 2 regions
   // Add Tier 1 or Tier 2 annotations to functionLines
-  const functionLines = want
+  const functionLines = backend
+    .allEndpoints(want)
     .filter((fn) => fn.minInstances)
```

Comment:
nit* `fn` => `e` (here and onwards)

Reply:
It's actually against our prettier according to my plugin, but I like it better. I'll manually change and investigate later